lib/util/validationResponse.js

```diff
@@ -84,9 +85,7 @@ exports.mapper = {
   'DUPLICATE_OPERATIONID': 'M6004',
   'MISSING_PATH_PARAMETER_DEFINITION': 'M6005',
   'EMPTY_PATH_PARAMETER_DECLARATION': 'M6006',
-  'MISSING_PATH_PARAMETER_DEFINITION': 'M6007',
   'EQUIVALENT_PATH': 'M6008',
-  'DUPLICATE_PARAMETER': 'M6009',
```

Comment:
Why did we remove these constants from the mapper?

Reply:
@amarzavery  because those 2 are duplicates. Not sure if the mapped values are used anywhere with a hard dependency so I didn't adapt the numbers for remaining ones. Let me know if I should

Reply:
@vishrutshah - There were some duplicates here. Do you know what is the effect of these error codes in reporting?

Reply:
@amarzavery  I don't thing right now they are used for anything since we don't have official documentation from them also I don't thing @vishrutshah  spends his time on Github anymore :)